BlitzBoard/Features/Repository/HomeScreenRepository.swift

```diff
@@ -0,0 +1,27 @@
+// LeagueTableRepository.swift
+// BlitzBoard
+//
+// Created by Kiasha Rangasamy on 2024/04/11.
+
+import Foundation
+
+// MARK: Typealias
+typealias LeagueTableResult = (Result<[LeagueTableModel], APIError>) -> Void
+
+// MARK: LeagueTableRepository Protocol
+protocol LeagueTableRepositoryType: AnyObject {
+    
+    func fetchLeagueTableResults(completion: @escaping(LeagueTableResult))
+}
+
+// MARK: Classes
+class LeagueTableRepository: LeagueTableRepositoryType {
+    
+    private let apiHandler = APIHandler()
+    
+    func fetchLeagueTableResults(completion: @escaping (LeagueTableResult)) {
+        
+        let url = EndPoint.leagueTableEndpoint
```

Comment:
apiHandler.request(endpoint: EndPoint.leagueTableEndpoint, method: "GET", completion: completion)
 
 remove let url = EndPoint.leagueTableEndpoint. It remove a line of code which is always a win